atlp-rwanda/cod-be
# #181414599 Feature: User registration with JWT

**What does this PR do?**
This PR implements a feature to register a user and gives a JWT (token) on success registration

**Description of Task to be completed**
  User registration with JWT
  
**How should this be manually tested?**
Navigate to the terminal and run the following commands to ensure to correct setup of the project:
npm install: To install required packages
npm run createDb: to create a database as configured in the project environment variables file
npm run dbMigrate: To migrate all required tables in the database
npm run dev: To start the project server
Use the APP_URL and add /api-docs to visit the documentation and test the feature from there

Story Link : https://www.pivotaltracker.com/story/show/181414599
